### PR TITLE
Removed Dashboard link from navbar for non-admin users

### LINKED ISSUE #397: USER STORY: Remove dashboard link on nav bar
This is a user story!

1. Please use the following format for the title:

USER STORY: <Remove dashboard link on nav bar>

### PR DESCRIPTION
**Pull request by gitbiruk2010**
This is a pull request!

Please use the following format for the title:
< https://github.com/SeattleColleges/belindas-closet-nextjs > _ < 7 > _ < #397  > _ < pr or issue name >

Please fill out this description with the following:
Resolves #397
Description of what this pull request is for.
Additional information to help others that are testing.
Screenshots/gifs/videos/diagrams/other visual aids.
Remember to:
Add reviewers, assignees and labels in the menu on the right. --->
Enable auto-merge (squash) at the bottom of the page.

### CODE REVIEW

**Review by Asfand00 — APPROVED** [review]
I'm currently logged in as user on this image and this is what I see:
dashboard_navbar

When logged in as an admin I see the dashboard link so this works on my end (Ignore the credentials in the sign in form, it's my autofill at work, I'm using the admin@gmail.com account)
dashboard_admin

File: components/Navbar.tsx [hunk 1]
```diff
@@ -10,8 +10,14 @@ import CategoryDropDownMenu from "./CategoryDropDownMenu";
...
 const drawerWidth = 240;
-const navItems = ["Home", "Sign In", "Donation", "Mission", "Dashboard", "Contact"];
-const links = ["/", "/auth/sign-in", "/donation-info", "/mission-page", "/dashboard", "/contact-page"];
+
+// Removed "Dashboard" from navItems array
+const navItems = ["Home", "Sign In", "Donation", "Mission", "Contact"];
+// const navItems = ["Home", "Sign In", "Donation", "Mission", "Dashboard", "Contact"]; // Original
+
+// Removed "/dashboard" from links array
+const links = ["/", "/auth/sign-in", "/donation-info", "/mission-page", "/contact-page"];
```
Comment (miqo1385) [inline components/Navbar.tsx:19]: I was dealing with similar problem when sign in on nsc-event nextjs, i also think I solved. you can go to nsc-event  and branch is called Navbar updates #346 and compare. maybe that will be helpful

Comment (gitbiruk2010) [reply components/Navbar.tsx:19]: @miqo1385 . Thanks. i will check that out.

**Review by keiffer213 — APPROVED** [review]
This looks great! I believe this was initially just to remove the dashboard button all together since it will be accessible by creators and admin on their account page, but this is great as well!

When I'm logged in as a regular user:
image

When I'm logged in as a creator/admin, the dashboard shows up, but when I log out it disappears:

https://github.com/SeattleColleges/belindas-closet-nextjs/assets/54779894/115557d5-2f08-4e1f-b5f2-299871bc6728

**Review by brinkbrink — APPROVED** [review]
Awesome work, @gitbiruk2010! Like @keiffer213 said, originally this was just to remove the dashboard button from the navbar altogether but this works just as well and since @keiffer213 is the product owner and he likes this, this is ready to merge.

**Pull request merged**